src/execution/execute.ts

```diff
@@ -234,10 +236,8 @@ function buildResponse(
 /**
  * Essential assertions before executing to provide developer feedback for
  * improper use of the GraphQL library.
- *
- * @internal
  */
-export function assertValidExecutionArguments(
```

Comment:
Can you please quickly look if someone is using those functions with removed "export"?
I usually use https://cs.github.com/ for that:
https://cs.github.com/?q=buildExecutionContext%20graphql%20language%3ATypeScript&scopeName=All%20repos&scope=#

It's not a reason to not remove "export" it's just if you see some popular library using it we need to contact the author at some point.

Reply:
I do not have access to that technology preview. I will revert the removed exports, and we can consider removing them in a separate PR.